src/Veldrid/Vk/VkTexture.cs

```diff
@@ -32,6 +32,7 @@ internal unsafe class VkTexture : Texture
         public override uint MipLevels { get; }
 
         public override uint ArrayLayers { get; }
+        public uint ActualArrayLayers => _actualImageArrayLayers;
```

Comment:
Not sure how best to handle this; I noticed there's some `effectiveArrayLayers` handling spread across a couple different places.  Is there somewhere common this can be moved that would be better for consolidation?

Reply:
I could see it moving into the base Texture class, but I'm not sure it would have the same kind of usefulness or meaning across all backends,or whether it would just introduce more confusion because of differences in cubemap/array layer handling.